Source/WebCore/platform/graphics/Font.h

```diff
@@ -226,6 +225,8 @@ class Font : public RefCounted<Font> {
     };
     const Attributes& attributes() const { return m_attributes; }
 
+    bool glyphIsColorGlyph(Glyph) const;
```

Comment:
Should we name this `isColorGlyph` for consistency with `isZeroWidthSpaceGlyph`?

Reply:
Good idea.